code/citadel/vore/eating/belly_vr.dm

```diff
@@ -138,12 +138,17 @@
 // The purpose of this method is to avoid duplicate code, and ensure that all necessary
 // steps are taken.
 /datum/belly/proc/nom_mob(var/mob/prey, var/mob/user)
-//	if (prey.buckled)
-//		prey.buckled.unbuckle_mob()
+//	if (prey.anchored)
+//		prey.anchored.unbuckle_mob()
 
 // Super super messy. prey.forceMove.owner doesn't work if there's no prey.
 	prey.loc = owner
-	internal_contents |= prey
+
```

Comment:
This check needs to be applied at the same check to see if someone can be taken.